user-guide/Basic_Functionality/Visio/linking_shapes/Linking_a_shape_to_the_alarm_state_of_a_Data_Display_page.md

```diff
@@ -19,3 +19,8 @@ For example, if you specify the following shape data, the shape will have the co
 | ---------------- | ------ |
 | Element          | IRD 01 |
 | DataDisplayPage  | Audio  |
+
+> [!NOTE]
+>
+> - The displayed alarm level corresponds to the alarm level of the full Data Display page and does not consider service context.
+> - We recommend using the shape data field of type *DataDisplayPage* judiciously, as it creates a subscription on all parameters of the page.
```

Comment:
@KlaasSL, I've made a few changes to the note. Is the content still OK?

Reply:
Yes, still looks good!